### PR TITLE
Fix after_destroy hooks for gemini and bitflyer

### PR DESCRIPTION
**Pull request by yachtcaptain23**
https://github.com/brave-intl/publishers/issues/3252
https://github.com/brave-intl/publishers/issues/3253

### CODE REVIEW

File: test/models/gemini_connection_test.rb [hunk 1]
```diff
@@ -20,6 +20,17 @@ class GeminiConnectionTest < ActiveSupport::TestCase
     end
   end
 
+  describe 'after destroy hook' do
+    let(:gemini_connection) { gemini_connections(:connection_with_token) }
+    let(:publisher) { publishers(:gemini_completed) }
+    test 'publisher no longer has a selected wallet provider' do
+      assert publisher.selected_wallet_provider, gemini_connection
```
Comment (tsmartt) [inline test/models/gemini_connection_test.rb:27]: Did you mean `assert_equal` or just wanted to `assert` multiple items? If the latter I think you'll need separate statements.

File: test/fixtures/publishers.yml [hunk 2]
```diff
@@ -56,7 +56,7 @@ gemini_completed:
   default_currency_confirmed_at: "<%= DateTime.now %>"
   subscribed_to_marketing_emails: true
   selected_wallet_provider: connection_with_token (GeminiConnection)
-
+
```
Comment (tsmartt) [inline test/fixtures/publishers.yml:59]: Space

Comment (yachtcaptain23) [inline test/fixtures/publishers.yml:59]:
```suggestion
```

File: test/models/gemini_connection_test.rb (hunk 1 above)
Comment (yachtcaptain23) [inline test/models/gemini_connection_test.rb:27]:
```suggestion
      assert_equal publisher.selected_wallet_provider, gemini_connection
```

Comment (yachtcaptain23) [reply test/models/gemini_connection_test.rb]: `assert_equal`. Thanks, I updated the test.

File: test/fixtures/publishers.yml (hunk 2 above)
Comment (yachtcaptain23) [reply test/fixtures/publishers.yml]: It should contain the space. Thanks

**Review by tsmartt — APPROVED** [review]

**Pull request merged**